helpdesk_mgmt/controllers/myaccount.py

```diff
@@ -10,9 +10,12 @@
 class CustomerPortalHelpdesk(CustomerPortal):
     def _prepare_portal_layout_values(self):
         values = super()._prepare_portal_layout_values()
-        partner = request.env.user.partner_id
-        ticket_count = request.env["helpdesk.ticket"].search_count(
-            [("partner_id", "child_of", partner.id)]
+        ticket_count = (
```

Comment:
This fix is incorrect. What has to be done is to protect the views adding the smart-button with the helpdesk group.

Reply:
But you approved it in https://github.com/OCA/helpdesk/pull/216

Reply:
Yeah, my bad, but it's not the right fix. Maybe I was tired of fighting everything :stuck_out_tongue_closed_eyes:

Reply:
Do you understand why is not the right fix?

Reply:
Fixed.